### ISSUE TITLE
Provide an Amazon linux 2 base image

### ISSUE THREAD

**Issue by dlespiau** [dlespiau/footloose#59]
It'd be nice to provide [Amazon Linux 2 footloose base images](https://github.com/dlespiau/footloose/issues/26#issuecomment-467624321). From a quick glance, it looks like a centos/rhel based image and version 2 comes with systemd.

Took the [official image](https://hub.docker.com/_/amazonlinux?tab=tags) for as spin and systemd isn't installed by default but it's only a `yum install systemd` away

Given the above, building an amazon linux 2 base image should be quite close to what we do for centos7: https://github.com/dlespiau/footloose/blob/master/images/centos7/Dockerfile. We should put it in `

A detail is that currently, only I can push images to `quay.io/footloose`, this should be automated and pushed from CI (#23). It doesn't prevent people from creating and using a new locally though.

**Comment by rothgar** [comment]
I tried to create a container locally but my version of `footloose` (0.1.0) says it doesn't have a `--image` flag. I'm assuming that's a feature on the master branch. Is there instructions on how to build the project with the latest binary?

**Comment by dlespiau** [comment]
That's correct, sorry about that! 3 ways to get around it

- Compile it yourself. Instructions are at : https://github.com/dlespiau/footloose#install
- Change the footloose.yaml file manually, you just need something like:

```
cluster:
  name: cluster
  privateKey: cluster-key
machines:
- count: 1
  spec:
    image: quay.io/footloose/amazonlinux2
    name: node%d
    portMappings:
    - containerPort: 22
```

- I can make a 0.2.0 release?

**Comment by dlespiau** [comment]
We now have the base image published based on #61, a follow-up PR (#62) documents that new value and adds end to end tests (including running docker in docker!). Considering this done :) Thanks for the idea and contribution!
